frontend/src/features/election_management/routes.tsx

```diff
@@ -0,0 +1,12 @@
+import { RouteObject } from "react-router";
+
+import { t } from "@/lib/i18n";
+
+import { ElectionReportPage } from "./components";
+
+export const electionManagementRoutes: RouteObject[] = [
+  {
+    index: true,
+    element: __API_MSW__ ? <div>{`${t("election.title.finish_data_entry")} - Abacus`}</div> : <ElectionReportPage />,
```

Comment:
This used to render the `NotAvailableInMock` component, but now just renders a `div` without further context. I don't care too much about MSW now that we have abacus-test.nl, but if we do it this way we should remove the `NotAvailableInMock` component entirely.